luigi/interface.py

```diff
@@ -41,6 +41,22 @@
 
 
 def setup_interface_logging(conf_file='', level_name='DEBUG'):
+    '''
+    Luigi logging can be configured in the following ways:
+
+        * Pass in the file name of a .ini style logging configuration.
+
+          The ``conf_file`` is passed to logging.config.fileConfig()
+
+        * Pass in a dictionary containing the logging configuation format as specified
+          in the `Python 3 documentation
+          <https://docs.python.org/3/library/logging.config.html#logging-config-dictschema>`_.
```

Comment:
This doc exists for python 2 as well. Feels weird to only call out Python 3. I'd either mention both or neither, explicitly.